build.gradle

```diff
@@ -2,17 +2,9 @@ buildscript {
   repositories { jcenter() }
 }
 
-plugins {
```

Comment:
That was the intention. It's one of the loose ends that I mentioned that need to be tied up. I was trying to strike a balance between getting some early feedback on the way I've structured the metrics code and not doing repetitive cleanup because there will be more iterations. I'll tinker around with gradle to see if I can move this into a gitignore'd build.local.gradle (or do you think it's worth looking into the core nebula version conflict issue at this point?).